### ISSUE TITLE
Disable autosave / draft save

### ISSUE THREAD

**Issue by Brossollet-Antonin** [typora/typora-issues#3996]
I'm working on documents saved on a veracrypt encrypted partition for privacy reason. 
Even after disabling autosave options, drafts are still saved in appdata if I'm not being extra careful. It would be great to have an option to disable altogether all autosave function including drafts. 
Thank you for this great software!

**Comment by glocalglocal** [comment]
And, like in Word, when you save or confirm you don't want to discard changes (ie when you exit gracefully), Typora should clean after itself and delete all drafts for that file. I just realised I had a year's worth of drafts since the first day I installed Typora, some of them potentially sensitive, and I didn't even know.

**Comment by zhaoqingqing** [comment]
how to Disable autosave ? 

in my typora is not work.

in settings i disable autosave，but in appdata folder also append new drafts xx.md

**Comment by pieczynski** [comment]
In Notepad++:

Remember: Create a backup file.

Open:
`C:\Users\{USER}\AppData\Local\Programs\Typora\resources\appsrc\window\frame.js`

Find:
`var t=window._options.userDataPath+(File.isWin?"\\":"/")+"draftsRecover"; `

Replace with:
`var t=window._options.userDataPath+(File.isWin?"\\":"/")+"!@#$%^&*()"; `

**Comment by shinegogo** [comment]
pieczynski's method works.

**Comment by Fealthas** [comment]
Yes I would also like a way to disable this or change the drafts folder to be next to the file.

**Comment by pieczynski** [comment]
The method above doesn’t work on Linux, so the easiest fix is to tweak the directory permissions.

First, locate the Typora configuration folder. Then, open a terminal and change the permissions for the "draftsRecover" directory for the root user.

`sudo chown -R root:root draftsRecover/
`
